### ISSUE TITLE
Document the change in ordering for Save, Save and Close, Cancel

### ISSUE THREAD

**Issue by RCheesley** [mautic/mautic-documentation#196]
- Save is now Apply
- Save and Close is now a green icon
- Order has been changed to improve UX

See PR's:

https://github.com/mautic/mautic/pull/9894
https://github.com/mautic/mautic/pull/9895

**Comment by fakela** [comment]
@RCheesley i wanted to work on this, I checked through the docs we didn't specifically go through the process of creating an email with images(screenshots) but this can be found in the  [create a thank you email](https://kb.mautic.org/knowledgebase/welcome-to-mautic/getting-started-with-mautic#3-create-a-thank-you-email)

So I will look towards updating on that end.  Loading up the PR to get the screenshots throws an error. Please, how do I fix that? 

I was suggesting we link the  [managing emails](https://docs.mautic.org/en/channels/emails/managing-emails#email-creation) section in the docs specifically the part that talks about email creation to the KB.

**Comment by RCheesley** [comment]
What error do you see when you open up the PRs?

I think it would be good to link the docs with the KB indeed!

**Comment by fakela** [comment]
@RCheesley Here is the screenshot of the error i get

<img width="1412" alt="Screenshot 2021-05-27 at 23 48 40" src="https://user-images.githubusercontent.com/39309699/119906417-1708ca00-bf46-11eb-933b-e50c8a7a0644.png">

**Comment by RCheesley** [comment]
Yes, you will get that on Mautibox if the PR has already been merged to features. Use the default branch rather than the PR and you should see Mautic with the PR applied in that case :)
